app/controllers/pages_controller.rb

```diff
@@ -57,6 +57,13 @@ def global_stats
     stats_arr.map { |s| Hashie::Mash.new(s) }
   end
 
+  def filter_faqs(faqs)
+    faqs.select do |faq|
+      stages = faq['Site Stage'].map(&:strip)
+      stages.include?('Main') || stages.include?('Pre Launch')
```

Comment:
So I'm noticing some duplicate questions between Main and Pre Launch (like `How do I get started`) – is there a way to deal with these as well, so they don't appear twice?

Reply:
Yeah, looks like `How do I get started` might be the only dupe. I'll add some logic to remove dupes and prefer the main version.